Britannia/localization/english/BTWK_buildings_l_english.yml

```diff
@@ -141,3 +157,31 @@
  building_type_antonine_wall_01_desc:0 "The Antonine Wall was constructed around a quarter of a century after Hadrian's Wall, during the reign of the Emperor Antoninus Pius. While it was intended to be a new, more northernly Caledonian frontier, it was occupied for only a few years before its desertion. "
  building_antonine_wall_01:0 "$building_type_antonine_wall_01$"
  building_antonine_wall_01_desc:0 "$building_type_antonine_wall_01_desc$"
+
+ ################
+ # BATH'S BATHS #
+ ################
+ building_type_btwk_aquae_sulis_baths_01:0 "Roman Baths"
+ building_type_btwk_aquae_sulis_baths_01_desc:0 "During the Roman conquest of Britain, the Romans built a series of baths around some natural springs from which the local Britons worshipped their God #italic Sulis #!(hence, the name of the settlment #italic Aquae Sulis#!). Over the following centuries during Roman occupation, this settlement would grow into a large resort complex, of which its baths being a staple feature."
+
+ building_btwk_aquae_sulis_baths_01:0 "Roman Baths"
+ building_btwk_aquae_sulis_baths_01_desc:0 "$building_type_btwk_aquae_sulis_baths_01_desc$ \n\nThe departure of Romans from Britain shifted of the focus of local polities to prioritising the security of their realms, rather than to the upkeep of amenities like this spa. As such, the once magnificent resort has fallen into disuse and decay, and stands more as a monument to an age long past."
+ building_btwk_aquae_sulis_baths_01:0 "Roman Baths - Restored"
+ building_btwk_aquae_sulis_baths_02_desc:0 "$building_type_btwk_aquae_sulis_baths_01_desc$ \n\nRenovations have been carried out on the spa resort to allow for the recreational bathing the Romans once enjoyed. This has encouraged a larger influx of freemen to bring their custom to the resort - a great boon to the local economy."
```

Comment:
"recreational" feels a bit too casual given how important baths were to the Roman way of life - I would probably lean into the societal aspect of the baths a lot more
In the same vein "freemen" is also kind of a weird word to be using here, I think - "citizen" might be better if you want to keep the wording